apps/llm-client/README.md

```
@@ -0,0 +1,96 @@
+
+```bash
+pnpm exec playwright install chromium   # first time only
+pnpm e2e
+```
+
+The E2E tests do **not** require a running llama-server — they mock the
+streaming endpoint via `page.route`.
+
+## Project layout
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)